extension_templates/dist_kern_panel.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+Extension template for pairwise distance or kernel on panel data.
+
+How to use this:
+- this is meant as a "fill in" template for easy extension
+- do NOT import this file directly - it will break
+- work through all the "todo" comments below
+- fill in code for mandatory methods, and optionally for optional methods
+- you can add more private methods, but do not override BaseEstimator's private methods
+    an easy way to be safe is to prefix your methods with "_custom"
+- change docstrings for functions and the file
+[- ensure interface compatibility by testing ??? no tests yet, fill this in later]
+- once complete: use as a local library, or contribute to sktime via PR
+
+Mandatory implements:
+    transforming    - _transform(self, X, X2=None)
+
+State:
+    none, this is a state-free scitype
+
+copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""
+
+from sktime.dists_kernels import BaseTrafoPw
```

Comment:
You imported tabular base class, not panel.

Reply:
yes, thanks, fixed it